tools/__tasks__/compile/javascript/index.js

```diff
@@ -1,12 +1,12 @@
-module.exports = [{
-    title: 'Compile JS',
+module.exports = {
+    description: 'Compile JS',
     task: [
         require('./clean'),
-        require('../inline-svgs/compile'),
+        'compile/inline-svgs',
```

Comment:
Yeah, I think it adds unnecessary API clutter. I would probably prefer to use `require` to import task objects, rather than strings to either execute tasks or terminal commands. This is just an opinion, though, not a showstopper 😄